### ISSUE TITLE
Add openbox support

### ISSUE THREAD

**Issue by ThanosApostolou** [oberon2007/vertex-theme#3]
@oberon2007 Since you are maintaining this theme yourself now, would it be easy for you to add openbox support similar to **vertex-maia-themes**?

**Comment by oberon-manjaro** [comment]
Good idea! All it needs would be to replace the current colour codes with their blue relatives.
Do you have a little time and would like to do that?
The relevant two files are just the **theme.rc**s [regular](https://github.com/manjaro/vertex-manjaro/blob/master/Vertex-Maia/openbox-3/themerc) and [dark](https://github.com/manjaro/vertex-manjaro/blob/master/Vertex-Maia-Dark/openbox-3/themerc) - a Light variation doesn't exist, yet. It would probably need some more investigation... :)

**Comment by ThanosApostolou** [comment]
I don't really have any idea about themes and colours but I guess I can give it a try in the weekend. Maybe it would be a good idea also to change minimize, maximize and close buttons to match the rest window managers' borders and the gtk3/gnome apps. However I don't know if this can be done easily, it will probably need more digging (I will need to find a good tutorial/guide for openbox themes if there is one).

**EDIT:** Aaaa I see that it's impossible to get the vertex minimize,maximize and close icons on xbm format needed for openbox. So, I will just try to change the colors.
